servers/audio_server.h

```diff
@@ -52,8 +52,8 @@ class AudioDriver {
 	uint64_t _last_mix_frames = 0;
 
 #ifdef DEBUG_ENABLED
-	uint64_t prof_ticks = 0;
-	uint64_t prof_time = 0;
+	SafeNumeric<uint64_t> prof_ticks{};
+	SafeNumeric<uint64_t> prof_time{};
```

Comment:
> I assume the underlying data might already be zero-initialized

Doesn't look like so, std::atomic's default ctor will leaves the atomic object in an uninitialized state , and SafeNumeric only has one explicit ctor.

Reply:
@akien-mga is right. Bear in mind that `explicit` is about type conversion (if not explicit, the constructor can be used transparently in type conversion to make an expression valid; otherwise, it must be _explicitly_ used). So, `explicit` doesn't mean that you have to make something extra for a constructor decorated with it to be called.

PS: The rest of the codebase doesn't use `explicit` much. However, it's a good hygiene measure in classes like this, where we favor explicitness: if using a `SafeNumeric`, don't let the code using it look like it's a primitive type. It can sound convenient, but hides key info to the reader.

Reply:
Thanks , I checked cpp's doc on explicit and you are right.